index.html

```diff
@@ -1,20 +1,320 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
 
     <!-- Bootstrap CSS -->
-    
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
 
-    <title>Hello, world!</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+
+    <!-- font awesome icons -->
+
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.css"
+      rel="stylesheet"
+      type="text/css"
+    />
+
+    <!-- Local CSS -->
+
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Hevons::Home</title>
+    <link rel="shortcut icon" href="./img/logo.png" type="image/x-icon">
+
+    <!-- Google material Icons -->
+
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Hello, world!</h1>
+  <body class="col-12">
+    <header>
+      <nav
+        class="navbar navbar-expand-lg navbar-light bg-light py-2 px-sm-4 shadow"
+      >
+        <div class="container-fluid">
```

Comment:
- [ ] Good job so far 👍 Kindly add the dark navbar as shown in the required design 
- Navbar:
![Capture](https://user-images.githubusercontent.com/85113567/181268478-4680c6de-b469-4a34-a4b4-6f26430340a8.PNG)

Reply:
Fixed